packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/App.kt

```diff
@@ -85,6 +85,15 @@ public interface App {
      */
     public fun allUsers(): Map<String, User>
 
+    /**
+     * Returns a wrapper for invoking App Services Application functions as the specified [user].
+     *
+     * [Atlas Functions documentation](https://www.mongodb.com/docs/atlas/app-services/functions/)
+     *
+     * @param user user with to execute the function.
+     */
+    public fun functions(user: User): Functions
```

Comment:
I know we had these on `App` in Java, but we started to move things that required a user to `User` instead so you didn't had to pass in the user all the time, then you would access this through `app.currentUser.functions` instead.

Reply:
Much better 👍